noteModel.js

```diff
@@ -0,0 +1,27 @@
+const mongoose = require('mongoose');
+const ObjectId = mongoose.Schema.Types.ObjectId;
+
+const NoteSchema = new mongoose.Schema({
+    _id: {
+       type: ObjectId,
+       required: true,
+       index: true,
+    },
+    title: {
```

Comment:
I think you'll want to add `index: true` to the title/body fields if you choose to add search features to your front end later on.